Core.BackgroundJobs/Model/ReadModels/ScheduleItSystemUsageOverviewReadModelUpdates.cs

```diff
@@ -94,6 +99,24 @@ private int HandleOrganizationUnitUpdated(CancellationToken token, int updatesEx
             return updatesExecuted;
         }
 
+        private int HandleOrganizationUpdated(CancellationToken token, int updatesExecuted, HashSet<int> alreadyScheduledIds)
+        {
+            foreach (var update in _updateRepository.GetMany(PendingReadModelUpdateSourceCategory.ItSystemUsage_Organization, BatchSize).ToList())
+            {
+                if (token.IsCancellationRequested)
+                    break;
+
+                using var transaction = _transactionManager.Begin(IsolationLevel.ReadCommitted);
+                var systemIds = _itSystemRepository.GetByRightsHolderId(update.SourceId).Select(x => x.Id).ToList();
+
+                var systemUsageIds = _itSystemUsageRepository.GetBySystemIds(systemIds).Select(x => x.Id);
+
```

Comment:
husk også at søge i read model repo (kræver tilføjelse af id for rightsholder + index i readmodel), Ellers virker det ikke med slettede organisationer

Reply:
Det er nok kun at søge i read model repo når vi får id på. Alle nye usages bliver jo dannet fra ny alligevel.